media/css/firefox/firstrun/firstrun.scss

```diff
@@ -0,0 +1,142 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+$break-large: 1305px;
+
+@import '../../../protocol/css/includes/lib';
+
+
+//* -------------------------------------------------------------------------- */
+// Common elements & variables
+
+html {
+    height: 100%;
+    min-height: 100vh;
+}
+
+section .mzp-l-content, div.mzp-c-hero-image, .browser-logo img {
+    display: inline-block;
+    max-width: 320px;
+    text-align: left;
+}
+
+.agreement {
+    padding-bottom: $spacing-sm;
+}
+
+.browser-logo img {
+    display: block;
+    max-width: 250px;
+    padding: $spacing-lg 0 $spacing-xl;
+}
+
+.field {
+    margin: 0;
+    padding: 0;
+}
+
+.mzp-c-button {
+    width: 100%;
+}
+
+.mzp-c-hero-desc {
+    padding-bottom: $spacing-md;
+}
+
+.mzp-c-hero-image {
+    margin-top: $spacing-sm;
+}
+
+.mzp-c-hero-title {
+    @include text-display-md;
```

Comment:
After looking again at the mockup I think this should bump up a notch to `text-display-lg` and you don't need to re-declare the size at other breakpoints, that's all baked into the mixin.